_posts/2016-09-22-foreman-community-newsletter---september-2016.md

```diff
@@ -0,0 +1,115 @@
+  usual plugin and media roundup
+---
+
+### Foreman 1.13 RC1 released
+
+Foreman 1.13 RC1 (and RC2 very soon!) is now available for general testing -
+and this is a big one!  We've got the initial work on IPv6 (addressing and
+partial orchestration), UEFI bootloader support, improvements to DHCP subnet
+parsing, Compute Resource upgrades (notably OpenStack Liberty), and support for
+Facter 2's structured facts - and that's just the headlines!
+
+With so many new features, your help is needed more than ever to test this
+release and ensure it meets the rock-solid quality expected of our releases.
+You can find out more and get involved from the [announcement][1_13_rc1] and
+please do report any [bugs][bugs]. Thanks!
+
+[1_13_rc1]: https://groups.google.com/d/topic/foreman-announce/klyCElDqal0/discussion
+[bugs]: http://projects.theforeman.org/projects/foreman/issues/new
+
+### 1.12.3 released
+
+The latest point release of the Foreman 1.12 series is now available, bringing
+bugfixes and support for Puppet Server 2.6. You can read all the details in the
+[announcement][1_12_3]
+
+[1_12_3]: https://groups.google.com/d/topic/foreman-users/0njkhkpRWzM/discussion
+
+### EL6 deprecated in 1.13, to be removed in 1.14
+
+Users of EL6 / Centos6 should be aware that 1.13 will be the last release with
+packages for this OS. 1.13 will be supported until early next year, and users
+are strongly encouraged to start planning their migrations before that date.
+
+Note that installation of EL6 hosts from Foreman is still supported - it's the
+packages for Foreman itself that will no longer be produced for EL6.
+
+### Katello 3.1 and 3.2 RC1
+
+The long-awaited Katello 3.1 release happened on September 9th, bringing a
+welcome round of improvements. You can learn more via the
+[announcement][K_3_1].
+
+Hot on 3.1's heels, Katello 3.2 RC1 has just been released - notably bring
+support for Puppet 4 to Katello. As always, your testing is hugely appreciated
+in making these releases as stable as possible - if you'd like to try it out,
+check out [this thread][K_3_2_rc1] and [let us know][kbugs] how it goes.
+Thanks!
+
+[K_3_1]: https://groups.google.com/d/topic/foreman-users/jnvGiYYpd9A/discussion
+[K_3_2_rc1]: https://groups.google.com/d/topic/foreman-users/3gGorphv1vI/discussion
+[kbugs]: http://projects.theforeman.org/projects/katello/issues/new
+
+### New media and blogs
+
+#### [Community Demo - 15th September](https://youtu.be/GFNPHypFhl4)
+
+Another packed demo, great to see them building up again. This time we had some
+Roles/Orgs updates, Ansible & Remote Execution integration, Katello
+improvements and a new MS-DHCP provider for the smart proxy. Check it out!
+
+### Upcoming events
+
+#### [AnsibleFest Brooklyn - 11th October](https://www.ansible.com/ansiblefest)
+
+We're hoping to have a small presence at the next AnsibleFest in Brooklyn next
+month - if you're planning to go along, [get in touch][mail_greg] and if
```

Comment:
Indeed, I asked him to go :) - but not everyone would be comfortable with be singled out for contact in a public newsletter, so lets get permission first